.github/scripts/download_assets.sh

```diff
@@ -1,46 +1,39 @@
 #!/bin/bash
 set -eu -o pipefail
 
-realpath() {
-    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
-}
-
-# path to the directory where this file is
-SELF_DIR_PATH="$(dirname "$0")"
-
-# a parameter for the destination of the assets can be passed.
-# the default is the directory assets, we assume the script is in .github/scripts/
-DATA_DIR="${1:-$SELF_DIR_PATH/../../assets}"
-DATA_DIR=`realpath $DATA_DIR`
-
+DATA_DIR="$PWD/../../assets"
 CHECKSUM_FILE="sha256sums"
+BASE_URL="http://s3-de-central.profitbricks.com/xayn-yellow-bert"
 
 download()
 {
-  NAME="$1"
-  VERSION="$2"
-  ARCHIVE_BASENAME="${NAME}_$VERSION"
-  ARCHIVE_NAME="$ARCHIVE_BASENAME.tgz"
-  TMP_ARCHIVE_NAME="$ARCHIVE_NAME.tmp"
-  URL="http://s3-de-central.profitbricks.com/xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
-
-  if [  -f "$DATA_DIR/$ARCHIVE_NAME" ]; then
-    echo "skip downloading $DATA_DIR/$ARCHIVE_NAME"
-  else
-
-    curl "$URL" -o "$DATA_DIR/$TMP_ARCHIVE_NAME" -C -
-    mv "$DATA_DIR/$TMP_ARCHIVE_NAME" "$DATA_DIR/$ARCHIVE_NAME"
-
-    cd "$DATA_DIR"
-    tar -zxf "$ARCHIVE_NAME"
-
-    # check content
-    cd "$ARCHIVE_BASENAME"
-    shasum -c "$CHECKSUM_FILE"
-  fi
+    ARCHIVE_BASENAME="$1_$2"
+    ARCHIVE_NAME="$ARCHIVE_BASENAME.tgz"
+    TMP_ARCHIVE_NAME="$ARCHIVE_NAME.tmp"
+
+    if [ -f "$DATA_DIR/$ARCHIVE_NAME" ]; then
+        echo "skip downloading $DATA_DIR/$ARCHIVE_NAME"
+    else
+        curl "$BASE_URL/$1/$ARCHIVE_NAME" -o "$DATA_DIR/$TMP_ARCHIVE_NAME" -C -
+        mv "$DATA_DIR/$TMP_ARCHIVE_NAME" "$DATA_DIR/$ARCHIVE_NAME"
+
+        cd "$DATA_DIR"
+        tar -zxf "$ARCHIVE_NAME"
+
+        # check content
+        cd "$ARCHIVE_BASENAME"
+        shasum -c "$CHECKSUM_FILE"
+    fi
```

Comment:
:wrench: 
I don't see any advantage or relevant change in this code w.r.t. to the previous one.

Reply:
I don't know why that change was made and i don't have the time to look into this. maybe we can revert those changes and move them into a separate pr?